stylus/cozy-ui/build.styl

```diff
@@ -1848,7 +1848,7 @@ Display an chip that represents complex identity
     Markup:
     <div class="{{modifier_class}}" style="display: inline-block; width: 10rem; height: 2rem; border: 1px solid black;"></div>
 
-    Weight: 14
+    Weight: 15
```

Comment:
Is this change normal?

Reply:
yup the `Weight` parameter is here to define a specific order in the styleguide, I just cleaned it since it wasn't right.